sudoku-app/src/App.js

```diff
@@ -92,13 +94,44 @@ function App() {
 }
 
 function Header() {
+  return(
+    <h1 class="Header">
+      Sudoku App
+    </h1>
+  )
+}
+
+function Buttons() {
```

Comment:
- Seems like there is already a Buttons function below!
- There are 2 return statements
- Potential fixes: move lines 105-120 into 136-137, add onClick events to the existing return statement